docs/ml_copies.md

```diff
@@ -70,3 +70,70 @@ The Replacement Capability is the ratio between the accuracy of the copy model w
 
 The Replacement Capability can also yield in some cases values much larger than one if the copy model generalizes better than the original. This is not normally the case, but it might happen if the original model was poorly chosen and the copy model family is better suited to describe the boundary profile of the original problem.
 
+----
+
+## Best practices
+
+
+ * **Determine motivation for the copy**
+
+Do you want to ensure you get the exact same behavior as the original model? There are many use cases where it is imperative to maintain the consistency between the original and the copy models. If this is so, use the empirical fidelity error to evaluate your copy.
+
+Do you only want to ensure you get a model with the performance as good or better as the original model but maintaining consistency is not important? In this case, use the copy replacement capability to evaluate your copy.
+
+
+### Model selection
+
+ * **Test data**
+ 
+If some labeled original test data is available, it is always more reliable to use that to compute the metrics and carry out the copy evaluation. Even if that original data is not labeled, it can always be labeled using the prediction of the original model and then be used to evaluate the copy. But this will at least ensure that the sampling is done following the original feature distribution.
+       
+If no original data is available, then a test synthetic dataset can be generated for evaluation of the copy. But be aware that, if this dataset is generated with the same sampler that was used to carry out the copy, the test will focus on the same regions where the copy has had more training. Hence, the fidelity or the performance of the copy will not be evaluated as rigorously on the outskirt decision boundary.
+
+ * **Sampling in evaluation**
+
+What does it mean if the copy has a low empirical fidelity error with synthetic data, but at the same time has a high fidelity error with test data? 
+
+The copy is often created without having a priori an idea of the distribution of the real data, this means that real data may be located mostly in a particular region while synthetic data will tend to be all over the place. This means that the original model which has been trained with real data will have a stronger emphasis on having certain boundaries right, while the copy model will be trained with data that enforces to have a similar boundary as the original mostly everywhere.
```

Comment:
I don't think it is necessarily a higher resolution. This would happen only if data is covering more parts of the feature space in one case than in the other. What I meant was that a higher number of samples in a certain region will force the algorithm to be "right" there (because of a higher weight in the loss function) and perhaps be more flexible about being wrong elsewhere.

Reply:
Agreed, that's what I meant by "higher resolution" :)